contribs/gmf/less/layertree.less

```diff
@@ -75,8 +51,15 @@
 
   .leaf,
   .group {
-    font-size: @font-size-small;
+    font-size: ceil(@font-size-base * 0.9);
```

Comment:
@font-size-base indeed comes from bootstrap.
It's set to 14px by default otherwize we decide to change it.